metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -94,12 +96,69 @@ object ServerCommands {
        |""".stripMargin
   )
 
+  val GotoLocation = Command(
+    "goto",
+    "Goto location",
+    """|Move the cursor to the definition of the argument symbol.
+       |
+       |Arguments: [string], where the string is a SemanticDB symbol.
+       |""".stripMargin
+  )
+
   /**
    * Open the browser at the given url.
    */
   val OpenBrowser: Regex = "browser-open-url:(.*)".r
   def OpenBrowser(url: String): String = s"browser-open-url:$url"
 
+  val GotoLog = Command(
+    "goto-log",
+    "Check logs",
+    "Open the Metals logs to troubleshoot issues."
+  )
+
+  val OpenIssue = Command(
+    OpenBrowser("https://github.com/scalameta/metals/issues/new/choose"),
+    "Open issue on GitHub",
+    "Open the Metals repository on GitHub to ask a question, report a bug or request a new feature."
+  )
+
+  val StarMetals = Command(
+    OpenBrowser("https://github.com/scalameta/metals"),
+    "Star Metals on GitHub",
+    "Give Metals some love and star it on GitHub 🙏"
+  )
+
+  val StarBloop = Command(
+    OpenBrowser("https://github.com/scalacenter/bloop"),
+    "Star Bloop on GitHub",
+    "Give Bloop some love and star it on GitHub 🙏"
+  )
+
+  val ChatOnGitter = Command(
+    OpenBrowser("https://gitter.im/scalameta/metals"),
+    "Chat on Gitter",
+    "Open the Metals channel on Gitter to discuss with other Metals users."
+  )
+
+  val ReadVscodeDocumentation = Command(
+    OpenBrowser("https://scalameta.org/metals/docs/editors/vscode.html"),
+    "Read Metals documentation",
+    "Open the Metals website to read the full instructions on how to use Metals with VS Code."
+  )
+
+  val ReadBloopDocumentation = Command(
+    OpenBrowser("https://scalacenter.github.io/bloop/"),
+    "Read Bloop documentation",
+    "Open the Bloop website to read the full instructions on how to install and use Bloop."
+  )
+
+  val FollowTwitter = Command(
+    OpenBrowser("https://twitter.com/scalameta"),
+    "Follow Scalameta on Twitter",
```

Comment:
Once again, following is one time action. Once followed, user will probably see it on twitter and this action would become misleading. 
`See Scalameta on Twitter` ?